### ISSUE TITLE
InsertRange Help :D

### ISSUE THREAD

**Issue by BryanSan** [jamesmontemagno/mvvm-helpers#21]
Hi James!. I would like to have a method to insert a collection, would you mind to give me some guidance of how can i help you to build such feature? Thanks :D

**Comment by weitzhandler** [comment]
i can do it, but i'd first wanna see my PR merged, check it out #26.

**Comment by weitzhandler** [comment]
Looks like this repo isn't maintained anymore, which is a bit of a shame, the `ObservableRangeCollection` is a very common use case.
@jamesmontemagno

**Comment by weitzhandler** [comment]
I can do it, but i'd first wanna see my PR merged, check it out #26.
Looks like @jamesmontemagno is busy, I hope he gets to do some maintenance around here a bit.

**Comment by taublast** [comment]
```
 //-------------------------------------------------------------
        public void InsertRange(int index, IEnumerable<T> collection)
        //-------------------------------------------------------------
        {
            NotifyCollectionChangedAction notificationMode = NotifyCollectionChangedAction.Add;

            CheckReentrancy();

            int currentIndex = index;
            var changedItems = collection is List<T> ? (List<T>)collection : new List<T>(collection);
            foreach (var i in changedItems)
            {
                Items.Insert(currentIndex, i);
                currentIndex++;
            }
            OnPropertyChanged(new PropertyChangedEventArgs("Count"));
            OnPropertyChanged(new PropertyChangedEventArgs("Item[]"));
            OnCollectionChanged(new NotifyCollectionChangedEventArgs(NotifyCollectionChangedAction.Add, changedItems, index));
        }
```
